src/commands/list.js

```diff
@@ -8,10 +8,17 @@ var Table = require("cli-table");
 var _ = require("lodash");
 var logger = require("../logger");
 
+/**
+ * This command is deprecated in favor of `projects:list` command
+ * TODO: Remove this command after 02/09/2020.
```

Comment:
Actually, would you remove this TODO? We're going to probably remove the command sooner than this, so we won't commit to a timeframe here.

Reply:
(Remove the date from `getProjects`, but leave the todo to remove it.)

Reply:
Done